cmd/serve.go

```diff
@@ -120,3 +135,53 @@ func serve(_ context.Context) {
 		logger.Fatal("failed to run server", zap.Error(err))
 	}
 }
+
+func getAuthMiddleware(ctx context.Context, config fositex.OAuth2Configurator) (echo.MiddlewareFunc, error) {
+	issuer := config.GetAccessTokenIssuer(ctx)
+
+	audience, err := url.JoinPath(issuer, "userinfo")
+	if err != nil {
+		return nil, err
+	}
```

Comment:
This audience is specific to the UserInfo endpoint. We can probably just remove this for now and revisit audience handling.

```suggestion
```

Reply:
gotcha, removed.